src/game-master/middlewares/phoenix-gm-middleware.js

```diff
@@ -60,6 +63,8 @@ function handleServerMessage(store, message) {
 
     switch (message.name) {
         case MESSAGE_NAME.gameMasterSessionState:
+            store.dispatch(setReadyState(message.sandboxStatus === ReadyState.READY));
```

Comment:
GM session state has `sandboxStatus` property

Reply:
it is handle below

Reply:
or am I missing smth

Reply:
My fault, stupid I am